src/main/resources/entity-types/inventory/simple_holdings_record.json5

```
@@ -798,13 +798,28 @@
         itemDataType: {
           dataType: 'objectType',
           properties: [
+            {
+              name: 'holdings_note_type_id',
```

The holdings_note_type_id refers to the UUID of the Holdings Note Type. However, I believe we should have a field labeled "Holdings Note Type" that displays a dropdown of available note types.
```suggestion
              name: 'holdings_note_type',
```